cmd/influxd/run/server_test.go

```diff
@@ -5713,12 +5712,13 @@ func TestServer_ContinuousQuery(t *testing.T) {
 	}
 
 	// Wait for CQs to run. TODO: fix this ugly hack
-	time.Sleep(time.Second * 5)
+	//time.Sleep(time.Second * 5)
```

Comment:
Thought about it but if the skipped test below gets fixed, this my need to be put back in. Thought it would be better to leave it as a reminder. In the mean time, better not to slow the test down.

Maybe this delay can be removed altogether if we have a way to check query status now?